frontend/src/app/libs/common-components/shared/variables.scss

```diff
@@ -0,0 +1,78 @@
+@mixin text-view($font-family, $font-size, $letter-spacing, $line-height) {
```

Comment:
not exactly, i mean make all .class mixins, co consumer will be able to include such mixin to it's own classes

Reply:
sorry, changed